django-stubs/contrib/auth/models.pyi

```diff
@@ -34,7 +35,7 @@ class GroupManager(models.Manager[Group]):
 class Group(models.Model):
     objects: ClassVar[GroupManager]
 
-    name = models.CharField(max_length=150)
+    name: models.CharField[str | int | Combinable, str]
     permissions = models.ManyToManyField(Permission)
```

Comment:
... however I can't explicitly annotate `permissions: models.ManyToManyField[..., ...]` as the second type var is solved by the plugin by a class that doesn't exist in stubs and at runtime

Reply:
You can have a look here how it can be done: https://github.com/typeddjango/django-stubs/pull/2214/files#diff-0236ca9e909bc4743fa03e6653fc33adbaaef02a34d8698501a63a59ad624f1fR42-R51